.nvmrc

```diff
@@ -0,0 +1 @@
+v18
```

Comment:
I guess it doesn't truly matter, but what is your position on this? Why not 16 or 20?